src/app/services/xml-parser-service/xmp-parser.service.ts

```diff
@@ -117,14 +121,50 @@ export const ContentParser = (content: any): string => {
   } else if (
     content instanceof org.cru.godtools.shared.tool.parser.model.Multiselect
   ) {
-    // console.log('CONTENT: Multiselect');
-    return '';
+    // console.log('CONTENT: multiselect')
+    return 'multiselect';
+  } else if (
+    content instanceof org.cru.godtools.shared.tool.parser.model.Flow
+  ) {
+    // console.log('CONTENT: Flow');
+    return 'flow';
   } else {
-    console.log('CONTENT: Unknown');
+    // console.log('CONTENT: Unknown');
     return '';
   }
 };
 
+interface DimensionParser {
+  type: string;
+  symbol: string;
+  value: number;
+}
+export const DimensionParser = (dimension: any): DimensionParser => {
+  let type = null,
+    symbol = null,
+    value = null;
+  if (
+    dimension instanceof
+    org.cru.godtools.shared.tool.parser.model.Dimension.Percent
+  ) {
+    type = 'percent';
+    symbol = '%';
+    value = dimension.value;
```

Comment:
you'll probably want to do `value = dimension.value * 100` since you will want to present `0.5` as `50%`